habitat_sim/simulator.py

```diff
@@ -553,45 +553,7 @@ def draw_observation(self) -> None:
                  (has it been detached from a scene node?)"
             )
 
-        # get the correct scene graph based on application
-        if self._spec.sensor_type == SensorType.SEMANTIC:
-            if self._sim.semantic_scene is None:
```

Comment:
Do we need to add an ESPCHECK for this in the C++ or are we good?

Reply:
@Skylion007 : ESP_CHECK is only used in the C++, NO?

Reply:
Yeah, but this check was removed. I just want to make sure we throw the error somewhere.

Reply:
OK, good catch. I will see if I can do something in the `getActiveSemanticSceneGraph`.